src/mumble/mumble.pro

```diff
@@ -446,12 +446,11 @@ unix {
   HAVE_PULSEAUDIO=$$system(pkg-config --modversion --silence-errors libpulse)
   HAVE_PORTAUDIO=$$system(pkg-config --modversion --silence-errors portaudio-2.0)
 
-  !isEmpty(HAVE_PORTAUDIO):!CONFIG(no-portaudio) {
+  !isEmpty(HAVE_PORTAUDIO):CONFIG(portaudio) {
```

Comment:
This doesn't make any sense. If CONFIG(config)... add it again? :-)

We can drop this whole section.

Drop HAVE_PORTAUDIO.
Drop this check.

Reply:
I read it as "Enable PortAudio" instead of "Add flag"...